### PR TITLE
docs: for xds bootstrap

### PR DESCRIPTION
**Pull request by arkodg**
Fixes: https://github.com/envoyproxy/gateway/issues/31

### CODE REVIEW

File: docs/latest/user/egctl.md [hunk 1]
```diff
@@ -451,12 +453,176 @@ spec:
           kind: Service
           name: backend
           port: 3000
-          weight: 1
+EOF         value: /thPrefix
```
Comment (zirain) [inline docs/latest/user/egctl.md:456]: is this right?

Comment (arkodg) [reply docs/latest/user/egctl.md]: thanks for catching this, fixed it, ptal

File: docs/latest/user/egctl.md [hunk 2]
```diff
@@ -459,6 +461,175 @@ spec:
 EOF
 ```
 
+You can see the output contains a [EnvoyProxy](https://gateway.envoyproxy.io/latest/api/config_types.html#envoyproxy) resource that
+can be used as a starting point to modify the xDS bootstrap resource for the managed Envoy Proxy fleet.
+
+```
```
Comment (zirain) [inline docs/latest/user/egctl.md:467]:
```suggestion
```yaml
```

File: docs/latest/user/egctl.md (hunk 1 above)
Comment (arkodg) [reply docs/latest/user/egctl.md]: hoping https://github.com/envoyproxy/gateway/pull/1175/files lands soon

Comment (zirain) [reply docs/latest/user/egctl.md]: only available for github internal project

**Review by zirain — APPROVED** [review]

**Review by Xunzhuo — APPROVED** [review]

**Pull request merged**